View3D/Services/MeshOptimizerService.cs

```diff
@@ -103,6 +123,31 @@ public static MeshObject CreatedReducedCopy(MeshObject original, float factor)
             clone.RebuildVertexBuffer();
 
             return clone;
+        }      
+
+        private static MeshObject GetReducedMeshSimplygon(ISimplygon sg, MeshObject originalMesh, float factor)
+        {
+            // -- Copy MeshObject data into Simplygon containers
+            SimplygonHelpers.InitSGGeometryDataObject(sg, out var pGeometryData, originalMesh);
+            SimplygonHelpers.FillSGVertices(ref pGeometryData, originalMesh);
```

Comment:
Dont use ref unless needed, in this case it is not as you are not working with value objects